### PR TITLE
Podcast Detail storyboard / headerview refactor

### PR DESCRIPTION
**Pull request by berkmolla**
Made the PodcastDetailViewController into a storyboard VC to make it more designer/iOS developer friendly, and got rid of 1/4 of the code in the HeaderView class so fewer chance of bugs, easier to change, etc. 

Also bumped deployment target up to 11.0 as XCode complains when putting a WKWebView in IB otherwise. iOS 11 adoption rate is around 87% and iOS 12 is coming in a month or two, still supported by the same devices as iOS 11 so should be fine hopefully.

### CODE REVIEW

**Comment by jasonify** [comment]
Hey, thanks for the PR. Even though we have a few Storyboards already in the codebase I think we actually want to move away from them.

The main problem  right now is that programatic layout code is being done with a library that doesn't seem to be working great for iPhone 10.

Also, ideally we could do pretty small PRs when possible. Just something to keep in mind.

**Comment by berkmolla-je** [comment]
Hey Jason - is there a place where these things are being discussed? Or some kind of guide that outlines the style / decisions that have been made about the codebase so that contributors are aware of these before they open PRs? 
And is there some kind of up to date task board so that contributors can see the features/enhancements that have been agreed to be useful and prioritised?

**Comment by berkmolla** [comment]
Hey Jason - is there a place where these things are being discussed? Or some kind of guide that outlines the style / decisions that have been made about the codebase so that contributors are aware of these before they open PRs?
And is there some kind of up to date task board so that contributors can see the features/enhancements that have been agreed to be useful and prioritised?

File: SEDaily-IOS/GeneralCollectionViewController.swift [hunk 1]
```diff
@@ -194,11 +195,14 @@ class GeneralCollectionViewController: UICollectionViewController {
     override func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
         if let viewModel = podcastViewModelController.viewModel(at: indexPath.row) {
             if let audioOverlayDelegate = self.audioOverlayDelegate {
-                let vc = PodcastDetailViewController(nibName: nil, bundle: nil, audioOverlayDelegate: audioOverlayDelegate)
-                vc.model = viewModel
-                vc.delegate = self
-            
-                self.navigationController?.pushViewController(vc, animated: true)
+                let sb = UIStoryboard(name: "PodcastDetail", bundle: .main)
+                let optionalVC = sb.instantiateInitialViewController() as? PodcastDetailViewController
```
Comment (deepjparekh) [inline SEDaily-IOS/GeneralCollectionViewController.swift:199]: `guard let podcastDetailViewController = sb.instantiateInitialViewController() as? PodcastDetailViewController else { 
return
} `
podcastDetailViewController.audioOverlayDelegate = audioOverlayDelegate
podcastDetailViewController.model = viewModel
podcastDetailViewController.delegate = self
navigationController?.pushViewController(podcastDetailViewController, animated: true)

**Pull request closed without merging**